frontend/appflowy_flutter/lib/workspace/application/tabs/tabs_bloc.dart

```diff
@@ -31,6 +34,10 @@ class TabsBloc extends Bloc<TabsEvent, TabsState> {
           emit(state.closeView(pluginId));
           _setLatestOpenView();
```

Comment:
I suggest the `ViewPB` parameter within the `_setLatestOpenView` should be the required one.

It would make more sense for `_setLatestOpenView(null);` to indicate clearing the latest opened view instead of the `_setLatestOpenView()`.

Reply:
It doesn't clear the latest open view though, what it does is that it infers the latest open view instead of just setting it to the view provided.